website/pages/en/querying/graphql-api.mdx

```diff
@@ -49,6 +49,23 @@ When querying a collection, the `orderBy` parameter may be used to sort by a spe
 }
 ```
 
+#### Example for nested entity sorting
+
+Sorting on the basis of nested entities is useful if you are looking to sort entities based on child fields.
```

Comment:
This is a bit of a circular sentence - I might make it more of a statement of fact:
"As of Graph Node version 0.30.0, entities can be sorted by fields on nested entities. In the following example..."